src/common/home/Home.jsx

```diff
@@ -132,6 +154,25 @@ const Home = () => {
           </Link>
         </div>
       </section>
+      <section className="home-tweets">
```

Comment:
Can all these classes be replaced with tailwind?

Reply:
might be but i don't know tailwind